sos/report/plugins/openstack_sahara.py

```diff
@@ -52,13 +52,14 @@ def postproc(self):
         ]
         connection_keys = ["connection"]
 
+        join_con_keys = "|".join(connection_keys),
```

Comment:
Dangerous copy&paste typo, please remove the trailing comma (I know how it happened :), just let build the f-string regular expression for this code to see the problem..).

Reply:
oops